.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
+# . "$(dirname "$0")/_/husky.sh"
```

Comment:
?

Reply:
No idea what that is.....

Reply:
It's commenting out the precommit hook via husky that is active (or should be) on this repo.

Reply:
Oh sorry, what i meant is that i dont know why that is there or has happened. I have not done that, ever.

Reply:
Ah right. I think @nilubava accidentally did in https://github.com/prisma/docs/pull/2170